clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -75,28 +85,31 @@ object ElasticsearchIndex extends ModelAutoDerivation {
     DateTimeFormatter.ofPattern("yyyy/MM/dd")
 
   /** Implicit summoner, for convenience. */
-  def apply[Document: ElasticsearchIndex]: ElasticsearchIndex[Document] =
-    implicitly[ElasticsearchIndex[Document]]
+  def apply[Index]: ElasticsearchIndex[Index] =
+    implicitly[ElasticsearchIndex[Index]]
 
-  implicit val WgsUbam: ElasticsearchIndex[DocumentWgsUbam] =
-    new ElasticsearchIndex[DocumentWgsUbam](
+  implicit val WgsUbam: ElasticsearchIndex[WgsUbamIndex.type] =
+    new ElasticsearchIndex(
       "wgs-ubam",
+      WgsUbamIndex,
       ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords
```

Comment:
If the mapper is the same for all, does it need to be provided as an argument?

Reply:
Probably not, but I'd prefer to get rid of that as part of a task for deprecating the other mapper.